c/driver/common/utils.h

```diff
@@ -44,7 +48,7 @@ struct StringBuilder {
   size_t capacity;
 };
 void StringBuilderInit(struct StringBuilder* builder, size_t initial_size);
-void StringBuilderAppend(struct StringBuilder* builder, const char* value);
+void StringBuilderAppend(struct StringBuilder* builder, const char* fmt, ...);
```

Comment:
IIRC, there's a compiler-specific attribute you can apply to make the compiler apply warnings like it does for printf & friends